validating operators/validating operators.cpp

```diff
@@ -0,0 +1,98 @@
+#include<iostream>
+#include <string> 
+
+using namespace std;
+
+/*
+	This function checks if the current token is a operator or not
+	this can be checked by an amazing obervation about operators.
+	operators are mostly of length 1 or 2.
+	lets consider others as the special case.
+*/
+bool isValidOperator(string s){
+	int n = s.length();		//calculating length of the operator
+	/*
+	the special case operators.
+	*/
+	if(s=="sizeof()"||s=="<<="||s==">>=")return true; 
```

Comment:
why not move this to before line 13. it will remove unnecessary declaration of n in special cases.

Reply:
I normally put variables in the start to make it readable and it won't have any effect as well.